project/Build.scala

```diff
@@ -414,12 +414,6 @@ object DottyBuild extends Build {
         "org.scala-sbt" % "api" % sbtVersion.value % "test",
         "org.specs2" %% "specs2" % "2.3.11" % "test"
       ),
-      version := {
-        if (isNightly)
-          "0.1.1-" + VersionUtil.commitDate + "-" + VersionUtil.gitHash + "-NIGHTLY"
-        else
-          "0.1.1-SNAPSHOT"
-      },
```

Comment:
You mean `baseVersion`? If so, won't the resulting jars from snapshot publish drop `SNAPSHOT` from the artefact name? Also, perhaps more importantly - how do you publish nightlies with the hash and commit date?